packages/sdk/src/interaction/interaction.js

```diff
@@ -191,6 +202,15 @@ export const makeArgument = (arg) => (ix) => {
   ix.arguments[tempId].resolve = arg.resolve
   return Ok(ix)
 }
+export const makeMeta = (data) => (ix) => {
+  if (Object.keys(data).length > META_KEYS.length) {
+    return Bad(ix, "Wrong number of metadata keys")
+  }
+  Object.keys(data).forEach(key => META_KEYS.includes(key) || delete data[key])
```

Comment:
I like this scrubbing, maybe include a console.warn or something with link to the docs saying that something invalid was included and it has been scrubbed, and a link to valid fields

Reply:
Will add to next PR